### PR TITLE
Small enhancement and Bug Fixes

### PR DESCRIPTION
**Pull request by AverageJosh**
Adding more prompt information
Writing out a RC command after selecting branches, in the case a new RC has to be made and you don't need to go through the prompt again (happened to me last week)
Fixing issue with only 1 branch being selected and RCI breaking

### CODE REVIEW

**Review by tonymishler — APPROVED** [review]
lgtm

**Comment by AverageJosh** [comment]
image

File: git-rci.ps1 [hunk 1]
```diff
@@ -46,7 +46,10 @@ function Select-Branch {
 
         Write-Host ""
         Write-Host "Filter: '$filterText'" -ForegroundColor Cyan
-
+        Write-Host ""
+        Write-Host "'Use Arrow Keys to navigate. [Enter] to select/deselect the branch. [Esc] to complete"
+		
+        # If testing locally on Powershell, switch IncludeKeyDown => IncludeKeyUp
         $key = $host.UI.RawUI.ReadKey("NoEcho,IncludeKeyDown")
```
Comment (mdekrey) [inline git-rci.ps1:53]: Key presses so often have this kind of problem. I wonder if there's a different way that's more resilient 🤔

**Review by mdekrey — APPROVED** [review]
Looks great to me. Thanks!

**Pull request merged**